src/contexts/filterReducer.js

```diff
@@ -1,13 +1,18 @@
 const filterReducer = (state, action) => {
 switch (action.type) {
+
     case "Max Safety":
```

Comment:
Please discuss this name here you should provide a name here like this 
MAX_SAFETY, 
in case all variables should be capital. Please discuss this.

Reply:
Thanks for the suggestion Rahul. Will definitely fix this, when i'm cleaning the code. Thanks for pointing it out